feat: Toolbox - major version bump (#3520)
neo4j/graphql: .changeset/hip-fireants-kiss.md

---
"@neo4j/graphql-toolbox": major
---

feat: Toolbox - major version bump to version 2.0.0

A high-level overview of changes made:

- All the editors are now powered by [CodeMirror](https://codemirror.net/) version 6 and support all the features and functions from the previous version.
- On the query editor page we've introduced tabs to store several queries and mutations. Improved the solution for resizing the different editors (query, variables, and response).

> - The favorites on the type definitions page did get an update. It's now possible to order favorites and download them individually.

Reviewer: ```suggestion
- The favorites feature on the type definitions page has been updated. It's now possible to order favorites and download them individually.
```

---
"@neo4j/graphql-toolbox": major
---

> feat: Toolbox - major version bump to version 2.0.0

Reviewer: Is this good enough? Do we not need to add context for why it's now 2.0.0?

Author: Nah it's not 😃 . I'll extend this.
